pkg/queue/manager.go

```diff
@@ -112,6 +115,9 @@ type Manager struct {
 
 	admissionFairSharingConfig *config.AdmissionFairSharing
 	secondPassQueue            *secondPassQueue
+
+	penaltyMap             *PenaltyMap
```

Comment:
```suggestion
	afsEntryPenalties             *AfsEntryPenalties
```
It is preferred in golang not to add sufix indicating the type (I know in some languages it is).